README.md

```diff
@@ -76,7 +85,7 @@ This strategy can be found in `packages/chrome-extension/src/fingerprint/strateg
 
 **Note:** Implementation of both of these strategies is not perfect, ideally they should also handle time-out scenarios and connection issues with the website. It was decided to leave it out to keep it simple.
 
-### Development
+## Development
 
 After cloning the repository perform these operations:
 
```

Comment:
Add the Support and feedback (optional) and Licence sections to the end of the Readme file.

Reply:
Done! I've added both sections